src/App.js

```diff
@@ -5,17 +5,26 @@
 */
 
 // Import the state hook
-import React from 'react';
+import React, { useState } from 'react';
 // Import the Posts (plural!) and SearchBar components, since they are used inside App component
 // Import the dummyData
 import './App.css';
+import Posts from './components/Posts/Posts.js';
+import SearchBar from './components/SearchBar/SearchBar.js';
+import dummyData from './dummy-data.js';
 
 const App = () => {
   // Create a state called `posts` to hold the array of post objects, **initializing to dummyData**.
   // This state is the source of truth for the data inside the app. You won't be needing dummyData anymore.
   // To make the search bar work (which is stretch) we'd need another state to hold the search term.
-
+  const [posts, sPosts] = useState(dummyData);
```

Comment:
Try to keep the setter function as 'setPosts' Usually, one letter variables, functions etc, is not great practice and can be confusing to future developers looking at your code. Plus the 'set' keyword is most commonly used.